src/administration/integrations/notifications.md

```diff
@@ -24,9 +26,41 @@ Health notifications can be set up via the [Platform.sh CLI](/gettingstarted/cli
 
 ### Email notifications
 
+#### Default email notifications
+
+When your project is new or when your project doesn't have a health notification set up, Platform.sh creates a default low-disk email notification for all [Project Admins](/administration/users.md).
+
+You can check out the default email notification by running `platform integration:get`.
+
+```bash
+platform integration:get
++--------------+---------------+
+| Property     | Value         |
++--------------+---------------+
+| id           | abcdefghijklm |
+| type         | health.email  |
+| role         |               |
+| from_address |               |
+| recipients   | - '#admins'   |
++--------------+---------------+
+```
+
+The following `recipients` values are available:
+
+* `#admins` maps to all project admins and up.
+* `#viewers` maps to everyone with access to the project.
```

Comment:
These shortcuts are a new feature.  They should be included in the main setup for email notifications, not tucked in here for the defaults.